pfcpiface/up4.go

```diff
@@ -128,7 +130,40 @@ func (m meter) String() string {
 }
 
 func (up4 *UP4) addSliceInfo(sliceInfo *SliceInfo) error {
-	log.Errorln("Slice Info not supported in P4")
+	//FIXME: UP4 currently supports a single slice meter rate common between UL and DL traffic. For this reason, we
+	//  configure the meter with the largest slice MBR between UL and DL.
+	if !up4.isConnected(nil) {
+		return fastpathDisconnected
```

Comment:
We generally use more generic errors defined in `errors.go`. I suggest using `ErrOperationFailedWithReason`.